src/frontend/scss/objects/_course_glimpses.scss

```diff
@@ -68,8 +68,7 @@ $course-glimpse-content-padding-sides: 0.7rem !default;
     @include sv-flex(1, 0, calc(25% - #{$r-course-glimpse-gutter * 2}));
   }
 
-  &:hover,
-  &:focus {
+  &--focused {
```

Comment:
Instead to toggle a class focused through javascript event listeners, what do you think to keep the `:hover` then set `pointer-events` to none for the whole course-glimpse then set `pointer-events` to `fill` for interative children (`a` and `button`?)

```scss
pointer-events: none;

a, button {
  pointer-events: fill;
}

&:hover, &:focus { ... }
```

Reply:
Hey, that's pretty clever :) great idea.

Reply:
So I'm using your technique, it's great!

One small detail that can feel weird is that in order to be able to mouse hover the icons having titles, I need to activate pointer-events on them too, so that makes the card activate its shadow in this case too.

For keyboard, the idea was to activate the card shadow when focusing anything in the card *and* outline as usual the link that is actually focused inside it. Hence the usage of "focus-within" on the card wrapper, and "focus-visible" instead of "focus" on the title link, to prevent triggering the focus styles while just clicking with the mouse.